app-arch/unshield/unshield-1.4.2.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Tool and library to extract CAB files from InstallShield installers"
+DESCRIPTION="Unshield extracts CAB files from InstallShield installers, \
+used to install software on Microsoft Windows based machines."
+HOMEPAGE="https://github.com/twogood/unshield"
+COPYRIGHT="2017 David Eriksson"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/twogood/unshield/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="5dd4ea0c7e97ad8e3677ff3a254b116df08a5d041c2df8859aad5c4f88d1f774"
+SOURCE_FILENAME="unshield-$portVersion.tar.gz"
+PATCHES="unshield-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	unshield$secondaryArchSuffix = $portVersion
+	cmd:unshield$secondaryArchSuffix = $portVersion
+	lib:libunshield$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	unshield${secondaryArchSuffix}_devel = $portVersion
+	devel:libunshield$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	unshield$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:c++$secondaryArchSuffix
+	cmd:cc$secondaryArchSuffix
+	cmd:cmake
+	cmd:make
+	"
+
+TEST_PREREQUIRES="
+	cmd:diff
+	cmd:find
+	"
+BUILD_PREREQUIRES+="$TEST_PREREQUIRES"
+
+BUILD()
+{
+	cmake . \
+		$cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_EXE_LINKER_FLAGS="-liconv"
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	install -d -m 755 "$docDir" "$manDir" "$dataDir/expander/rules"
+	install -t "$docDir" ChangeLog
+
+	mv "$dataDir/man/man1" "$manDir/man1"
+	rm -rf "$dataDir"
```

Comment:
Should be: rm -rf "$dataDir/man"

I asked yesterday also, why the qoutes here (and in other places mentioning $paths)?